bin/inx-chronicle/config.template.toml

```diff
@@ -4,7 +4,7 @@ username = "root"
 password = "root"
 
 [api]
-port = 9092
+port = 8042
```

Comment:
Arbitrary change?

Reply:
It was colliding with the Rest API of Hornet's `private_tangle`.